client/db/src/utils.rs

```diff
@@ -327,6 +327,23 @@ pub fn read_db<Block>(
 	})
 }
 
+/// Remove database column entry for the given block.
+pub fn remove_db<Block>(
```

Comment:
I see that this naming follows the `read_db` function above, but it is really confusing to me IMHO.

Isn't this more a `remove_block_from_db` or similar?

If I understand this correctly, this will remove a block entry from  the db?

Reply:
It is generic and may remove anything that is referenced with `col_index`. E.g. justifications. I'll rename it to `remove_from_db`